.github/workflows/ci.yaml

```diff
@@ -818,8 +818,8 @@ jobs:
             -qq \
             --timeout=9 \
             --durations=10 \
-            -n auto \
-            --cov homeassistant.components.${{ matrix.group }} \
+            -n 0 \
```

Comment:
Python 3.10 current has issues in our CI caused by xdist in combination with... something :(

Disabling xdist at least mitigates the issue for now.

We also had issues with Python 3.9, for which the root cause was in our base images. That has been fixed separately buy replacing our CI containers that are used to run pytest in.